proj/src/main/webapp/static/signup.html

```diff
@@ -55,18 +41,17 @@ <h4>welcome</h4>
           <div class="row justify-content-md-center">
             <form>
               <label for="fname">first name:</label><br>
-              <input type="text" id="fname" name="fname" class="input"><br>
+              <input type="text" id="fname" name="fname" class="input" required><br>
               <label for="lname">last name:</label><br>
-              <input type="text" id="lname" name="lname" class="input"><br>
+              <input type="text" id="lname" name="lname" class="input" required><br>
```

Comment:
It looks like First Name and Last Name are not being used when the sign up button is clicked. Should they be?

Reply:
Nevermind I see it in the other PR.